documentation/docs/12-adapters.md

```diff
@@ -52,7 +52,7 @@ Most adapters will generate static HTML for any [prerenderable](/docs/page-optio
 +import adapter from '@sveltejs/adapter-static';
 ```
 
-You can also use `adapter-static` to generate single-page apps (SPAs) by specifying a [fallback page](https://github.com/sveltejs/kit/tree/master/packages/adapter-static#spa-mode).
+You can also use `adapter-static` to generate single-page apps (SPAs) by specifying a [fallback page](https://github.com/sveltejs/kit/tree/master/packages/adapter-static#spa-mode) and disabling SSR as page option (see [`SSR and SPA`](/docs/page-options#ssr-and-spa))
```

Comment:
this link is invalid without https://github.com/sveltejs/kit/pull/7275, so these two PRs should be a single PR. even if you're using the github UI you can still edit multiple files on the same PR. you just need to edit your branch (e.g. https://github.com/Tommertom/kit/tree/patch-1) rather than the sveltekit `master` branch

can you combine these PRs so we can discuss and review holistically?